src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,12 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireInlineCoding = () => {
+  const sol_file_Regex = /http:\/\/\S+.sol/g;
```

Comment:
I believe the coding convention for this repository is `camelCase`.  This variable should be `solFileRegex`.